server/shared/configFetcher.go

```diff
@@ -8,6 +8,7 @@ import (
 	"time"
 
 	"github.com/rs/zerolog/log"
+	"github.com/woodpecker-ci/woodpecker/server/plugins/configuration"
```

Comment:
sort import

Reply:
what tooling / guideline are you using for the import sorting ? I usually use vscode with the languageserver as default setup...

Reply:
Ok I think I got it, main point is the grouping into std / external / woodpecker right ?

Reply:
I am using VS-Code as well. For import sorting we use golangci-lint, if that's what you were asking for.

Reply:
Yep, I have that setup as linting tool too :-)